mypy-requirements.txt

```diff
@@ -2,4 +2,4 @@
 typing_extensions>=3.10
 mypy_extensions>=0.4.3
 typed_ast>=1.4.0,<2; python_version<'3.8'
-tomli>=1.1.0; python_version<'3.11'
+tomli>=1.1.0; python_version<'3.12'
```

Comment:
We only want to install tomli on Python 3.11 if we are running tests, so this should only be added to `test-requirements.txt` for Python 3.11 and later. If you change `mypy-requirements.txt`, tomli will always be installed, even though it's unnecessary for running mypy (it's only needed for self check and compilation).